wip-mariocao-min-difficulty-dr.md

```diff
@@ -0,0 +1,213 @@
+
+    ```rust
+    let k = 1.5 * (average_reputation - minimum_reputation);
+    let m = -k / (active_reputed_identities_length - 1);
+    ```
+
+3. The curve is used to assign the initial eligibility to the previously sorted identities (see _Step 3: triangle_ in Fig.2).
+
+4. The identities eligibility is offset by adding the remaining reputation points that were not redistributed in the previous step (see _Step 4: rectangle_ in Fig.2).
+
+    ```rust
+    let remaining_reputation = total_active_reputation - total_triangle_reputation;
+    let offset_reputation = remaining_reputation / (active_reputed_identities_length);
+    ```
+
+    This step is important to keep the same data request mining difficulty for newcomers or Sybil identities.
+
+5. The rest of the `offset_reputation` integer division is assigned (in order) to the sorted identities until depleted (see _Step 5: remaining_ in Fig.2).
+
+    ```rust
+    let identities_with_extra_eligibility = (remaining_reputation) % active_reputed_identities_length;
+    ```
+
+Steps 4 and 5 aim to preserve the sum of reputation equal to the sum of eligibility. Geometrically, this can be defined as matching the area of the trapezoid (Fig.2) with the area of the reputation curve (Fig.1).
+
+The identity's reputation, from the perspective of chain state, is not altered after applying the [RepPoE] algorithm.
+
+
+## Specification
+
+
+### Removal of the dynamic threshold
+
+**(1)**: The [Dynamic Threshold] factor MUST be removed from the _RepPoE_ algorithm.
+
+As eligibility across reputed identities is fairer due to the changes described in [witnet-rust#1496], the [Dynamic Threshold] does not have anymore a significant impact on the probability of mining data requests. Additionally, the [Dynamic Threshold] was not properly updated to use the eligibility metrics instead of the reputation scores.
+
+In case of abnormally low ARS sizes, it is assumed that there are in the order of thousands of non-reputed identities computing their eligibility to be selected to solved the data request. This assumption is in line with the minimum difficulty set in [WIP-0012] and previous network conditions documented in [WIP-0009].
+
+
+### Introduction of a minimum data request mining difficulty
+
+**(2)**: A lower bound on data request mining difficulty is proposed. For all data requests following the activation of this proposal, eligibility MUST be computed using the greater of two values: the ARS size and a constant `minimum_difficulty`:
+
+```rust
+let dr_eligibity = identity_eligibility / max(total_active_reputation, minimum_difficulty) * replication_factor
+```
+
+
+### Equate the data request minimum difficulty to the mining difficulty
+
+Analogously to [WIP-0012], such lower bound to be effective, its value needs to be strictly greater than `replication_factor`, as otherwise, ARS members will have an eligibility equal or greater than 100%.
```

Comment:
```suggestion
Analogously to [WIP-0012], for such lower bound to be effective, its value needs to be strictly greater than `replication_factor`, as otherwise, ARS members will have an eligibility equal or greater than 100%.
```